06.wc/wc.rb

```diff
@@ -8,9 +8,9 @@
 STDIN_COL_WIDTH = 7
 
 def main
-  params, argv_files = parse_argv
-  detail_files = argv_files.is_a?(Array) ? get_detail_files(argv_files) : [get_detail_stdin(argv_files)]
-  output(detail_files, params)
+  options, argv_files = parse_argv
+  detail_files = argv_files.empty? ? [read_detail_stdin] : get_detail_files(argv_files)
```

Comment:
read ~ stdinという名前は標準出力の値を読み取るだけ、という読めそうですが、実際はhashを返しますね。
read_detail_stdinがあまり大した仕事をしていないので、

```suggestion
  detail_files = argv_files.empty? ? [count_sentence($stdin.read)] : get_detail_files(argv_files)
```

とするのもアリかなーと思いました

Reply:
あと、detail_files だと「詳細なファイル」と読めるので、中身はFileオブジェクトかな？と思われそうです。
英語的には「ファイルの詳細＝file_details」の方がベターかもしれません

Reply:
修正しました。